src/Components/Address/AddressFormFields.tsx

```diff
@@ -139,6 +139,25 @@ export const AddressFormFields = <V extends FormikContextWithAddress>(
     return null
   }, [props.withLegacyPhoneInput, props.withPhoneNumber])
 
+  const phoneCountryOptions = useMemo(() => {
+    const hasValue =
+      values.phoneNumberCountryCode && values.phoneNumberCountryCode !== ""
+
+    if (!hasValue) {
```

Comment:
If there is no `phoneNumberCountryCode` this adds an empty county option to show user it is empty and needs to be fixed.

Reply:
Cool! non-blocking - This extra option could be a constant outside the component to make clear what we are adding.